packages/app/src/views/AddTag.vue

```diff
@@ -9,14 +9,14 @@
       </div>
     </header>
 
-    <div class="bg-yellow-400 w-full flex-grow">
+    <main class="bg-gradient-to-b from-primary to-secondary w-full flex flex-col flex-grow overflow-y-auto">
```

Comment:
```suggestion
    <main 
    class="bg-gradient-to-b from-primary to-secondary w-full flex flex-col flex-grow overflow-y-auto"
    >
```

another suggestion to prevent a lint error

Reply:
fixed by eb28638610c40063e4cc5334a26e3906d8e81875